Tests/Mailer/MailerTest.php

```diff
@@ -84,7 +83,7 @@ private function getMailer()
         return new Mailer(
             new Swift_Mailer(
                 new Swift_Transport_NullTransport(
-                    $this->getMockBuilder(Swift_Events_EventDispatcher::class)->getMock()
+                    $this->getMockBuilder('\Swift_Events_EventDispatcher')->getMock()
```

Comment:
This change is not required. Travis currently fails due to a memory error, not because of this.

Reply:
It is required. Check this build: https://travis-ci.org/FriendsOfSymfony/FOSUserBundle/jobs/411741460

Reply:
Which is quite strange because the test already passed on master branch.